utils/reviewsUtils.js

```diff
@@ -79,6 +71,5 @@ export async function getAverageRating(id) {
     averageRating = imdbRes;
     maxRating = 10;
   }
-
```

Comment:
This change just removes an empty line of code. Avoid making such unnecessary changes in order to keep PRs clean and easier to review.